microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -34,11 +34,20 @@ Uploads a package to update an existing draft submission of an add-on product.
 |---|---|
 | `productID` | Required.  The product ID of the product to which the package must be uploaded. |
 
-###### Request headers
+###### Request headers (v1)
 
-*  Required.  `Authorization: Bearer <auth token>`
+The following request headers are required:
 
-*  Required.  `Content-Type: application/zip`
+* `Authorization: Bearer <auth token>`
+* `Content-Type: application/zip`
+
+###### Request headers (v2)
```

Comment:
maybe the v2 section should be listed first, if we want to encourage use of it.
similar to making MV3 the default, leftmost tab instead of MV2.
@sivMSFT
cc @bhuvanapriyap @yvsubhash

Reply:
result: implemented tabsets.  left/default is v2.